modules/objdetect/misc/python/test/test_charuco_board.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import print_function
+
+import os, tempfile, numpy as np
+
+import sys
+sys.path.append("../../../../../doc/pattern_tools")
+import gen_pattern
+
+import cv2 as cv
+from svglib.svglib import svg2rlg
+from reportlab.graphics import renderPM
```

Comment:
I propose to move the import to the test body and print warning, if the import fails.